src/a_star_search/core.clj

```diff
@@ -30,3 +50,4 @@
 ; (time (-main))
 ; or
 ; time lein run
+; this is a test for committing to github
```

Comment:
You certainly don't want things like this to still be in the code when you "turn it in" (or submit a pull request).